datafusion/tests/test_dataframe.py

```diff
@@ -84,10 +84,11 @@ def test_select_columns(df):
 
 
 def test_filter(df):
-    df = df.select(
+    df = df.filter(column("a") > literal(2))\
```

Comment:
Move the filter before the select due to https://github.com/apache/arrow-datafusion/issues/5949